docs/configuration/configuration.md

```diff
@@ -3298,10 +3298,20 @@ sigv4:
   [ role_arn: <string> ]
 
 # Optional OAuth 2.0 configuration.
-# Cannot be used at the same time as basic_auth, authorization, or sigv4.
+# Cannot be used at the same time as basic_auth, authorization, sigv4, or azuread.
 oauth2:
   [ <oauth2> ]
 
+# Optional AzureAD configuration.
+# Cannot be used at the same time as basic_auth, authorization, or sigv4.
+azuread:
+  # The Azure Cloud. If blank, the default cloud considered is AzurePublic.
```

Comment:
Could you provide the options for clouds here?

Reply:
Added cloud options